staging/src/k8s.io/cloud-provider/controllers/node/node_controller_test.go

```diff
@@ -770,7 +768,7 @@ func TestGCECondition(t *testing.T) {
 		recorder:                  eventBroadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: "cloud-node-controller"}),
 		nodeStatusUpdateFrequency: 1 * time.Second,
 	}
-	eventBroadcaster.StartLogging(klog.Infof)
```

Comment:
I'd probably use t.Logf in all tests, if it's available (e.g., not in some framework).

Reply:
I fix to use `t.logf` in the tests.